pymonetdb/sql/connections.py

```diff
@@ -35,6 +40,11 @@ def __init__(self, database, hostname=None, port=50000, username="monetdb",
             autocommit (bool):  enable/disable auto commit (default: False)
             connect_timeout -- the socket timeout while connecting
                                (default: see python socket module)
+            binary (int):  enable binary result sets when possible if > 0 (default: 1)
```

Comment:
It used to be a boolean but this leaves the door open to future extensions: 0 = no binary, 1 = current binary, 2 = future improved binary, etc. This is consistent with what happens on the wire: during the handshake, the server currently announces `BINARY=1` but in the future it might report a higher number